README.md

```diff
@@ -36,14 +36,29 @@ Just create a project, and you’re good to go.
 
 **You’ll need to have Node >= 6 on your local development machine** (but it’s not required on the server). You can use [nvm](https://github.com/creationix/nvm#installation) (macOS/Linux) or [nvm-windows](https://github.com/coreybutler/nvm-windows#node-version-manager-nvm-for-windows) to easily switch Node versions between different projects.
 
-To create a new app, run a single command:
+To create a new app, you may choose one of the following methods:
 
+### npx
 ```sh
 npx create-react-app my-app
 ```
 
 *([npx](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b) comes with npm 5.2+ and higher, see [instructions for older npm versions](https://gist.github.com/gaearon/4064d3c23a77c74a3614c498a8bb1c5f))*
 
+### npm init
+
+Since npm of 6+ `npm init <name> <...args>` command may be used as an alias for `npx create-<name> <...args>`:
```

Comment:
Some of this information is unnecessary, can we make it more in line with the `npx` header?

### npm
```sh
npm init react-app my-app
```

*(npm init comes with npm 6+ and higher, see [instructions for older npm versions](https://gist.github.com/gaearon/4064d3c23a77c74a3614c498a8bb1c5f))*

Reply:
Do we even need this section if they have npm 5+?

Should we only recommend `npx` since it's available in both npm 5/6?

Reply:
I think it makes sense to recommend it as an alternative as long as we're clear that it's only available in npm 6+.